src/foundation.cs

```diff
@@ -4132,19 +4132,27 @@ interface NSError : NSSecureCoding, NSCopying {
 		[return: NullAllowed]
 		NSErrorUserInfoValueProvider GetUserInfoValueProvider (string errorDomain);
 
-#if XAMCORE_2_0 && IOS
+#if IOS || MONOMAC
 
 		// From NSError (NSFileProviderError) Category to avoid static category uglyness
 
 		[iOS (11,0)]
+		[Mac (10,15)]
 		[Static]
 		[Export ("fileProviderErrorForCollisionWithItem:")]
 		NSError GetFileProviderError (INSFileProviderItem existingItem);
 
 		[iOS (11,0)]
+		[Mac (10,15)]
 		[Static]
 		[Export ("fileProviderErrorForNonExistentItemWithIdentifier:")]
 		NSError GetFileProviderError (string nonExistentItemIdentifier);
+
+		[iOS (13,0)]
+		[Mac (10,15)]
+		[Static]
+		[Export ("fileProviderErrorForOutOfDateItem:")]
+		NSError fileProviderErrorForOutOfDateItem (INSFileProviderItem updatedVersion);
```

Comment:
```suggestion
		NSError GetFileProviderErrorForOutOfDateItem (INSFileProviderItem updatedVersion);
```

Sadly there's already a `GetFileProviderError` with the same signature, so that won't work.

Reply:
yeah, copy-paste mistake from me - after that same signature error
it should have been `GetFileProviderErrorForOutOfDateItem` like you suggested